javascript/javascript1/week2/homework.js

```diff
@@ -0,0 +1,126 @@
+//Formal fullname
+
+
+function getFullname(firstname,surname,useFormalName = false){ 
+    if (useFormalName){         
+        return `Lord.${firstname} ${surname}`
+    }else{
+       return `${firstname} ${surname}`
+    }
+    } 
```

Comment:
This is strangely formatted and makes it hard to see if the function has the closing `}` or not. Please, set up Prettier.